tracetools_launch/tracetools_launch/action.py

```diff
@@ -430,10 +446,15 @@ def execute(self, context: LaunchContext) -> List[Action]:
         )
         self._ld_preload_actions = self._get_ld_preload_actions(events_ust)
 
+        # Append '-init' to the session name if it is an init session
+        if is_init_session:
+            session_name += '-init'
```

Comment:
Related to my other comment about the name of the `is_init_session` parameter and the overall snapshot feature vs the init events use-case, I would make this:

```suggestion
        # Append '-snapshot' to the session name if it is a snapshot session
        if is_init_session:
            session_name += '-snapshot'
```

However, maybe `'-snapshot'` should be added by `lttng_impl.setup()`? This way, it wouldn't have to remove `-snapshot` from the session name.